src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -1,39 +1,29 @@
 @echo off
 setlocal
 
-set PROJECT_DIR=%~1
-set DOTNET_CMD=%~2
-set TOOLRUNTIME_DIR=%~3
-IF [%BUILDTOOLS_TARGET_RUNTIME%]==[] set BUILDTOOLS_TARGET_RUNTIME=win7-x64
+set DOTNET_CMD=%~1
```

Comment:
I would prefer we keep the arugments the same otherwise we will require everyone to update their usage of init-tools.cmd in their repo which I don't think it is worth it just to eliminate PROJECT_DIR which we may end up needing again in the future. Lets just keep letting them pass it.